README.md

```diff
@@ -126,7 +132,7 @@ mailer.send_mail
 - [ ] Complete test cases
 - [ ] Use nodemailer-mock
 - [ ] Support file attachments
-- [ ] Support `cc`, `bcc` and `reply to`
+- [ ] Support `reply to`
```

Comment:
Why is 'cc', 'bcc' removed lol?

Reply:
Removed as TODO list since the change is already to support cc and bcc.

Reply:
Ah, this is todo.. thought this is a feature list.